spec/lib/granite/context_proxy_spec.rb

```diff
@@ -0,0 +1,65 @@
+# spec: unit
+
+require 'granite/context_proxy'
+
+RSpec.describe Granite::ContextProxy do
```

Comment:
I am not sure why this class needs their own tests. Seems like implementation detail of action contexts. It could be maybe a private const?

Reply:
It's not a class, it's a module that's included in `Granite::Action` and `Granite::Projector`. The purpose of tests I assume (I haven't written most of them, git is not handling renames well) is to test this behavior in isolation.